PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -559,26 +559,25 @@ protected function wait_channel($channel_id, $timeout = 0)
                 }
                 continue;
 
-            } else {
```

Comment:
@videlalvaro i mean this else. Because when you use `return` the code is stopped at that point, so no else is needed